### ISSUE TITLE
rename config/root-phase0

### ISSUE THREAD

**Issue by fgiloux** [faroshq/tmc#1]
Low priority:

config/root-phase0 relates to the initialization steps in kcp. As TMC would now get installed as an overlay rather than being part of kcp core this naming may be misleading. We could use something like config/kcp instead. The rational: apiexports and apiresourceschemas are APIs defined in kcp.

**Comment by mjudeikis** [comment]
Good point :) Lets not loose this once we recover that was removed and rename :)

**Comment by mjudeikis** [comment]
Fixing in https://github.com/faroshq/tmc/pull/4
